CHANGELOG.md

```diff
@@ -8,33 +8,55 @@ This projects adheres to [Semantic Versioning](https://semver.org/) and [Keep a
 
 _No documentation available about unreleased changes as of yet._
 
-## [0.1.0] - 201y-mm-dd
+## [0.1.0] - 2013-10-20
```

Comment:
2013 ?

Reply:
This should be the release date? 😆 

I've put the date of the first commit to the repo 😄 

Should I leave this empty, or do we want to set a release date?

Reply:
It should be the release date, whether we want to set it yet is another matter ;-)